test/integration/newapp_test.go

```diff
@@ -1711,7 +1712,7 @@ func TestNewAppSourceAuthRequired(t *testing.T) {
 	}
 }
 
-func setupLocalGitRepo(t *testing.T, passwordProtected bool, requireProxy bool) string {
+func setupLocalGitRepo(t *testing.T, passwordProtected bool, requireProxy bool) (string, func()) {
 	// Create test directories
 	testDir, err := ioutil.TempDir("", "gitauth")
```

Comment:
The files that we're talking about, are created in the tests and don't contains any useful info and they can be easily re-created again. It means that they should be always removed, because they're test data. And also they should be removed on the level where they were created.

@stevekuznetsov this change don't make our tests perfect but at least it makes them better than they're now. Are you agree?